go.mod

```diff
@@ -0,0 +1,28 @@
+module github.com/libp2p/go-libp2p-rcmgr
+
+go 1.17
```

Comment:
This should be Go 1.16 (will be bumped to 1.17 by Unified CI once we drop 1.16 support).

Reply:
ok.

Reply:
done.